src/groupeddataframe/fastaggregates.jl

```diff
@@ -159,6 +159,7 @@ end
 function groupreduce!(res::AbstractVector, f, op, condf, adjust, checkempty::Bool,
                       incol::AbstractVector, gd::GroupedDataFrame)
     n = length(gd)
+    counts = nothing
```

Comment:
Have you checked that this doesn't affect performance? Normally I'd expect inference to work, but you never know...